src/main/java/baritone/pathing/calc/AStarPathFinder.java

```diff
@@ -40,9 +40,9 @@
  */
 public final class AStarPathFinder extends AbstractNodeCostSearch implements Helper {
 
-    private final Optional<HashSet<Long>> favoredPositions;
```

Comment:
Why would you replace an optional with a nullable lol

Reply:
This is probably fine because we're using this ``Optional`` as a parameter, which is not the intended use of the class. Alternatively, we could pass ``orElse(null)`` to the method that has the Optional parameter.

Reply:
Ok I take this one back, this is probably better